index.html

```diff
@@ -22,12 +22,12 @@
                     <li><a href="#" class="nav-link">About</a></li>
                     <li><a href="#" class="nav-link">Contact</a></li>
                 </ul>
-                <img src="./image/union-image.png" alt="Nav Bar" >
+                <img src="./image/union-image.png" alt="Navigation Icon" >
             </nav>    
         </header>    
         <section>
             <div>
-            <h2 class="developer-name">I’m Mariane Glad to see you!</h2>
+            <h1 class="developer-name">I’m Mariane Glad to see you!</h1>

```

Comment:
- [ ] Kindly be noticed that in the mobile size there is an extra space at the left of the page please fix that.

![Screenshot from 2022-07-19 21-45-30](https://user-images.githubusercontent.com/33752735/179835981-6fd425df-0a65-4612-a465-dc9ce0da0973.png)